WordPress/gutenberg
# Dataviews: Fix sticky table headers

Fixes https://github.com/WordPress/gutenberg/issues/58977.


https://github.com/WordPress/gutenberg/assets/846565/a5e266f0-a5b4-432a-ba55-e2b0efbee51a



## Testing Instructions
1. Open a data view like 'Manage all pages'
2. As you scroll notice the ~filter bar~, table headers, and pagination stick
3. Check other layouts (Grid / List) work as before 

~With this approach the filter controls are also sticky where previously it was only the table headers. I haven't found an approach that avoids this yet, but perhaps it's useful? We can discuss that and investigate alternative approaches in this PR.~
